apps/backend/src/infrastructure/services/upload/onchainPublisher/index.ts

```diff
@@ -4,93 +4,52 @@ import { safeCallback } from '../../../../shared/utils/safe.js'
 import { createTransactionManager } from './transactionManager.js'
 import { compactAddLength } from '@polkadot/util'
 import { nodesRepository } from '../../../repositories/objects/nodes.js'
-import { EventRouter } from '../../../eventRouter/index.js'
-import { MAX_RETRIES } from '../../../eventRouter/tasks.js'
-import { config } from '../../../../config.js'
 
 const logger = createLogger('upload:onchainPublisher')
 
 export const transactionManager = createTransactionManager()
 
-const REPUBLISH_DELAY = 10_000
+const publishNodes = safeCallback(async (cids: string[]) => {
```

Comment:
`safeCallback` will swallow errors which means using `publishNodes` inside `createHandlerWithRetries` won't lead to retries on node publish failures.

Reply:
You're right